GNUmakefile

```diff
@@ -1,5 +1,7 @@
 VERSION ?= 0.3.0
 PREFIX ?= /usr/local
+INSTALL ?= $(shell command -v ginstall 2>/dev/null || echo install)
```

Comment:
Ideally this `GNUmakefile` would just use the POSIX features, such as `mkdir` and `cp` instead of `install`.

Reply:
I'd make this `INSTALL ?= install` instead, if you want a variable for the binary name, then override it from command line. But I might change those to mkdir and cp..